examples/module/src/gizmo/my_gizmo.py

```diff
@@ -28,6 +28,18 @@ def new(cls, config: ComponentConfig, dependencies: Mapping[ResourceName, Resour
         gizmo.my_arg = config.attributes.fields["arg1"].string_value
         return gizmo
 
+    @classmethod
+    def validate_config(cls, config: ComponentConfig) -> List[str]:
+        # Custom validation can be done by specifiying a validate function like this one. Validate functions
+        # can throw `throw` errors that will be returned to the parent through gRPC. Validate functions can
```

Comment:
```suggestion
        # can throw `raise` errors that will be returned to the parent through gRPC. Validate functions can
```

Reply:
I don' think `raise` needs to be in backticks